### ISSUE TITLE
Sharing

### ISSUE THREAD

**Issue by oflak** [MajeurAndroid/Android-Applications-Info#21]
How can I share the infos the apps gathers?

**Comment by dhatch48** [comment]
Yes, please! Thank you for this app as it is very helpful. The only thing lacking for me was a way to export the list. I was using this to build a whitelist of apps and it would be very handy if you could export all with at least the uID.
